Day 8 Pybites Challenge: Cars

bryanfctobin opened the pull request:
Cars cars cars cars cars cars cars

bryanfctobin committed:
Day 8 Pybites Challenge: Cars

Cars cars cars cars cars cars cars

mmoga commented on 8.py:
```diff
@@ -0,0 +1,66 @@
+        # loops through v, which is the list containing specific models
+        for i in v:
+            # Checks if "Trail" is included in name: ref: https://stackoverflow.com/questions/3437059/does-python-have-a-string-contains-substring-method
+            if "Trail" in i:
```
What if the _t_ is uncapitalized?

bryanfctobin committed:
Update 8.py to account for lowercase "trail"

Matt is a good reviewer.  Honestly, I probably wouldn't have pressed on this, but I appreciate that he did.  He smells nice.

bryanfctobin commented:
@mmoga Updated commit with requested changes
